stable/pipelines/templates/ml-pipeline/ui/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.deployment.create }}
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "pipelines.name" . }}
+    chart: {{ template "pipelines.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "ui.name" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: ml-pipeline-ui
```

Comment:
same
Also use app + release for all matchLabel selectors in all deployments (or they might cross talk with components from other pipeline releases)